### PR TITLE
Version management and execution environment + labeled elements in edit tables (in property sheets)

### PR DESCRIPTION
**Pull request by cfsnook**
These changes sort out the versions and dependency based on API baseline tools.
We also decided to upgrade to Java 1.8

The first commit (Aug) adds support for labeled elements in property sheet edit tables.

### CODE REVIEW

File: ac.soton.eventb.emf.diagrams.refactor/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -15,6 +15,7 @@ Require-Bundle: org.eclipse.emf.common.ui;bundle-version="[2.6.0,3.0.0)",
  ac.soton.eventb.emf.core.extension.navigator;bundle-version="[4.1.1,5.0.0)",
```
Comment (tshoang) [inline ac.soton.eventb.emf.diagrams.refactor/META-INF/MANIFEST.MF]:
```suggestion
 ac.soton.eventb.emf.core.extension.navigator;bundle-version="[5.0.0,6.0.0)",
```

**Review by tshoang — CHANGES_REQUESTED** [review]
@cfsnook: The dependency from refactor on extension.navigator needs to be updated

File: ac.soton.eventb.emf.diagrams.refactor/META-INF/MANIFEST.MF (hunk 1 above)
Comment (cfsnook) [reply ac.soton.eventb.emf.diagrams.refactor/META-INF/MANIFEST.MF]: oops I forgot to commit that one. Have done it now.

**Review by tshoang — APPROVED** [review]
Looks good to me now

**Pull request merged**